src/interval.js

```diff
@@ -13,10 +13,14 @@ function validateStartEnd(start, end) {
   } else if (!end || !end.isValid) {
     return new Invalid("missing or invalid end");
   } else if (end < start) {
-    return new Invalid(
-      "end before start",
-      `The end of an interval must be after its start, but you had start=${start.toISO()} and end=${end.toISO()}`
-    );
+    if (Settings.throwOnInvalid) {
```

Comment:
For consistency, the other calls to `new Invalid` in this method should also throw.

Actually, the other `from` constructor of this class (`fromISO`) relies on `Interval.invalid` which does `throw` based on Settings.

`fromDateTimes` should do the same, with something along the lines of:
```
const validationError = validateStartEnd(builtStart, builtEnd)
if (validationError == null) return new Interval({ start: builtStart, end: builtEnd })
else return Interval.invalid(validationError)
```

Reply:
Yeah, what Gilles said is right. If you use `Interval.invalid()`, you'll get the `throw` for free there and you won't have to check the settings here.

Reply:
So intead of throwing the exception inside validateStartEnd, I should trhow it when the function is called in case there is an error?

Reply:
Yeah, please throw inside of `Interval.invalid()` so that it covers all callers of that, and then call that from here instead of constructing a new instance directly.

Reply:
I did this but according to my test I'm receiving an object and not an exception, I'm not sure why